testing/architecture-test/src/changes/archunit-store/internal-api-nullability.txt

```diff
@@ -1645,6 +1644,7 @@ Class <org.gradle.internal.build.event.types.DefaultWorkItemDescriptor> is not a
 Class <org.gradle.internal.build.event.types.NotFoundFileDownloadSuccessResult> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (NotFoundFileDownloadSuccessResult.java:0)
 Class <org.gradle.internal.buildevents.BuildExceptionReporter$ExceptionFormattingVisitor> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (BuildExceptionReporter.java:0)
 Class <org.gradle.internal.buildevents.BuildLogger> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (BuildLogger.java:0)
+Class <org.gradle.internal.buildevents.BuildLoggerFactory$1> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (BuildLoggerFactory.java:0)
```

Comment:
🤔 Could you add the annotation and quickly fix-up the class instead?

Reply:
This is not necessary and was due to the typo above.